packages/fast-check/package.json

```diff
@@ -8,17 +8,26 @@
     "./package.json": "./package.json",
     ".": {
       "require": {
+        "types@<5.7": "./lib/cjs/types57/fast-check.d.ts",
         "types": "./lib/cjs/types/fast-check.d.ts",
         "default": "./lib/cjs/fast-check.js"
       },
       "import": {
+        "types@<5.7": "./lib/types57/fast-check.d.ts",
         "types": "./lib/types/fast-check.d.ts",
         "default": "./lib/fast-check.js"
       }
     }
   },
   "module": "lib/fast-check.js",
   "types": "lib/types/fast-check.d.ts",
+  "typesVersions": {
+    "<5.7": {
```

Comment:
Shouldn't we add "*" as they do in https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#exports-is-prioritized-over-typesversions?

Reply:
Indeed looks great then